docs/deploying/infrastructure.md

```diff
@@ -12,28 +12,28 @@ This is managed using [builder](https://github.com/elifesciences/builder) using
 {project name}--{env}
 ```
 
-| Stack Name          | Description                                        |
-| ------------------- | -------------------------------------------------- |
-| elife-xpub--ci      | Runs tests as part of CI/CD                        |
-| elife-xpub--demo    | [Demo site](https://demo--xpub.elifesciences.org/) |
-| elife-xpub--prod    | [Production](https://xpub.elifesciences.org/)      |
-| elife-xpub--end2end | TODO                                               |
+| Stack Name          | Description                                         |
+| ------------------- | --------------------------------------------------- |
+| elife-xpub--ci      | Runs tests as part of CI/CD                         |
+| elife-xpub--staging | [Staging](https://staging--xpub.elifesciences.org/) |
+| elife-xpub--prod    | [Production](https://xpub.elifesciences.org/)       |
+| elife-xpub--end2end | TODO                                                |
 
 This project has 3 active environments, and a fourth that will be configured closer to the time of actually going live. The end2end environment This env will be as close as possible to the prod env. (e.g. RDS)
 
 # Steps used for creating new stack
 
 In this example we created a new environment for "prod"
 
-* Create new builder stack - VM, Sec Group, DNS etc ... called "elife-xpub--prod"
+- Create new builder stack - VM, Sec Group, DNS etc ... called "elife-xpub--prod"
 
-  * Done using ./bldr launch:elife-xpub,prod (then picks the saltstack up from the formula)
-  * For "prod" this still deploys a local postgres
+  - Done using ./bldr launch:elife-xpub,prod (then picks the saltstack up from the formula)
+  - For "prod" this still deploys a local postgres
 
-* Create New Jenkins pipeline: "prod-elife-xpub"
+- Create New Jenkins pipeline: "prod-elife-xpub"
```

Comment:
Jenkins pipelines do not follow environments, so this should be `Add to one of the two existing Jenkins pipelines)